src/routes/token/addToken.ts

```diff
@@ -1,7 +1,68 @@
 import { RequestHandler } from 'express';
+import SorobanClient from 'soroban-client';
+import dotenv from 'dotenv';
 
-const addTokenRoute: RequestHandler = async (req, res) => {
-  res.json({ ok: 'ok' });
+import token from '../../models/token';
+import { transactionBuild } from '../../utils/token/transactionBuild';
+import addTokenToDb from '../../utils/token/addTokenToDb';
+import returnObj from '../../utils/returnObj';
+
+dotenv.config();
+
+const server = new SorobanClient.Server(process.env.SOROBAN_SERVER);
+
+const fee = 100;
+
+const addToken: RequestHandler = async (req, res) => {
+  try {
+    const { tokenAddress } = req.body;
+
+    const findToken = await token.findOne({ address: tokenAddress });
+    if (findToken) {
+      return res.json(returnObj(false, 'token exist in database', null));
+    }
+
+    const accountAdmin = await server.getAccount(process.env.ADMIN_ADDRESS);
+    const contract: object = new SorobanClient.Contract(tokenAddress);
+
```

Comment:
remove the dotenv
use getServer
use getFee
use getAdmin
rename tokenAddress to token

rename findToken to isTokenExist

rename the message to 'Token already exists'

import { Contract }
change type to Contact instead of :object